bagit/reader.go

```diff
@@ -128,6 +128,23 @@ func (r *Reader) loadtagfile(name string) error {
 	return scanner.Err()
 }
 
+// Checksum returns a pointer to the Checksum struct of the given filername
+// culled from the manifests map
+// If one does not exist for the given file, it returns nil
+
+func (r *Reader) Checksum(name string) *Checksum {
+
+     // load the manifest files in from whereever
+
+     err := r.loadManifests()
```

Comment:
I wonder if a guard should be added inside loadManifests() so that the manifest files are parsed at most once.